helpers/scrape.js

```diff
@@ -1,10 +1,11 @@
+// seems to work using import. good to resolve any remaining issues.
+// Please only include the needed helper code now - not sure what is still being used
```

Comment:
Do you mean delete the Runner files that I used to test before we moved the functions to srcServer?